src/utils/index.js

```diff
@@ -40,3 +42,25 @@ export function dividers (num: number): Array<number> {
   }
   return divs
 }
+
+export function getNoDefaultConsoleErrorMessages (reason: string, vmName: string, hasRdp: boolean, defaultConsole: string): string {
+  let message
+  switch (reason) {
+    case EMPTY_CONSOLES_LIST:
+      message = hasRdp
+        ? msg.cantOpenNonRdpGraphicalConsoleEmptyList({ vmName })
+        : msg.cantOpenGraphicalConsoleEmptyList({ vmName })
+      break
+    case NO_DEFAULT_CONSOLE:
+      message = hasRdp
+        ? msg.cantOpenNonRdpGraphicalConsoleNoDefault({ vmName, defaultConsole })
+        : msg.cantOpenGraphicalConsoleNoDefault({ vmName, defaultConsole })
+      break
+    default:
```

Comment:
It seems to me that this default case is never reached. Can you explain when we'll get this error and why?

Reply:
> I'd rather the messages be selected in the place they're needed than be in this util function...

I did it thanks :)